### PR TITLE
ICU-21249 BRS68RC Version update and regenerate configure for v68

### PR DESCRIPTION
**Pull request by erik0686**
##### Checklist

- [X] Issue filed: https://unicode-org.atlassian.net/browse/ICU-21249
- [X] Updated PR title and link in previous line to include Issue number
- [X] Issue accepted
- [ ] Tests included
- [x] Documentation is changed or added

### CODE REVIEW

File: icu4j/build.properties [hunk 1]
```diff
@@ -4,9 +4,9 @@
...
-api.report.version = 67
-api.report.prev.version = 66
-release.file.ver = 67_1
-api.doc.version = 67.1
-maven.pom.ver = 67.1
+api.report.version = 68
+api.report.prev.version = 67
+release.file.ver = 68_1
+api.doc.version = 68.1
+maven.pom.ver = 68.1
```
Comment (daniel-ju) [inline icu4j/build.properties:11]: We'll want this to be the following for the Release Candidate (RC):
```
release.file.ver = 68rc
api.doc.version = 68 Release Candidate
maven.pom.ver = 68.1-SNAPSHOT
```

File: tools/cldr/cldr-to-icu/build-icu-data.xml [hunk 2]
```diff
@@ -47,10 +47,10 @@
         <property name="specialsDir" value="${basedir}/../../../icu4c/source/data/xml"/>
 
         <!-- Default value for ICU version (icuver.txt). Update this for each release. -->
-        <property name="icuVersion" value="67.1.0.0"/>
+        <property name="icuVersion" value="68.1.0.0"/>
 
         <!-- Default value for ICU data version (icuver.txt). Update this for each release. -->
-        <property name="icuDataVersion" value="67.1.0.0"/>
+        <property name="icuDataVersion" value="68.1.0.0"/>
```
Comment (daniel-ju) [inline tools/cldr/cldr-to-icu/build-icu-data.xml:53]: Thanks for updating this as well! :)

**Comment by daniel-ju** [comment]
Can you force-push your changes into a single commit with the commit message containing the ICU ticket number?
Something like "ICU-21249 BRS68RC Update version numbers and regenerate configure" (we can also update the PR title to match).

File: icu4j/build.properties (hunk 1 above)
Comment (erik0686) [reply icu4j/build.properties:11]: Oh I didn't know that :D Thanks Daniel

**Comment by daniel-ju** [comment]
Thanks Erik, LGTM!
It seems like there's a merge conflict with the `icudata.jar` file due to PR #1276. 
Would you mind rebasing to update your branch, and then regenerating the .jar files to resolve the conflict?

**Comment by erik0686** [comment]
@daniel-ju Done!

**Comment by jefgen** [comment]
Thanks @erik0686! :)

The changes in the PR mostly look good!

Can you please update both the commit message and the PR to have "BRS68RC" in the title though ?
It's not strictly required, but I find that it really does help to denote the start of the BRS tasks when looking back at the commit history in `git log`, or at the PRs on GitHub.

However, it looks like the Java build is failing with the most recent force-push though.

From the logs:
```
 /home/vsts/work/1/s/icu4j/main/classes/core/build.xml:37: Error while expanding /home/vsts/work/1/s/icu4j/main/shared/data/icudata.jar
 java.util.zip.ZipException: archive is not a ZIP archive
```

Taking a quick look:
It appears that the new `icudata.jar` file is only 316 bytes. However, it should be more like ~12 MB instead. (It should be quite large since it contains the ICU data).

Can you check that it was regenerated okay without any errors?
We'll need to re-generate it and force-push another update again.

**Comment by erik0686** [comment]
@jefgen Yes! I am not exactly sure what happened but I rebuilt the solution and ran the script to generate the jar files and there were changes, so I believe maybe I did something wrong there somewhere. But I believe I have fixed it now :) I'll also update the messages!

**Comment by jefgen** [comment]
FYI: The failing Travis CI check (for the missing copyright headers) is due to PR #1265.

**Pull request merged**